src/Utilities/Microsoft.Build.Utilities.csproj

```diff
@@ -209,15 +209,20 @@
     </Compile>
     <!-- Win32 RC Files -->
     <RCResourceFile Include="native.rc" />
+    
```

Comment:
okay, gonna scan for spurious white-spaces. I should enable white-space visibility as well :)